notebooks/functions/load_zones.py

```diff
@@ -0,0 +1,10 @@
+import geopandas as gpd
+import sqlite3
+
+def load_zones(project):
+    
+    sql = "SELECT zone_id, population, Hex(ST_AsBinary(GEOMETRY)) geometry FROM zones;"
+    zones  = gpd.GeoDataFrame.from_postgis(sql, project.conn, geom_col="geometry", crs=3857)
+    zones.to_crs(3857, inplace=True)
```

Comment:
Por que 3857?  Pra estar em metros?  Se for por isso,  então a conversão tem que ser dentro da função de RAI